README.md

```diff
@@ -97,78 +55,61 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Every user can track their budge their will be no conflict]**
+- **[Able to add a category and in the category you add purchase payment]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://budget-app-385w.onrender.com)
```

Comment:
- [ ] You did a nice job but your live link is not same as the app running locally, kindly fix this